modules/run-compliance-scan.adoc

```diff
@@ -36,7 +36,10 @@ If any of the associated check fails for a control, the entire control state is
 
 .Procedure
 
-. Navigate to the RHACS portal and open the compliance dashboard by selecting *Compliance* from the navigation menu.
+. Navigate to the {product-title-short} portal and open the compliance dashboard by selecting *Compliance* from the navigation menu.
+. Optional: By default, information under all standards is displayed in the compliance results. To view only information from specific standards, perform the following steps:
+.. Click *Manage Standards*.
+.. Select the specific standards you want to display, and then click *Save*. Standards that are not selected do not appear in the dashboard display (including widgets), compliance results tables that are accessed from the dashboard, and PDF files created using the *Export* button. However, all default standards are included when results are exported as a CSV file.
```

Comment:
Because the default is selected, you can consider pro and con to emphasize the usual case will be to clear (whichever verb you use) check boxes to hide scan results for specific standards the you do not want to display.